### PR TITLE
[I18N] [16.0] pos_*: improve translation vi

### PR DESCRIPTION
**Pull request by locdang8**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/loyalty/i18n/vi.po [hunk 1]
```diff
@@ -280,12 +280,12 @@ msgstr ""
 #. module: loyalty
 #: model_terms:ir.ui.view,arch_db:loyalty.loyalty_report
 msgid "<span>Minimum purchase of</span>"
-msgstr "<span>Minimum purchase of</span>"
```
Comment (tranngocson1996) [inline addons/loyalty/i18n/vi.po:284]: @locdang8 phần này k dịch à mà lại xóa đi em

File: addons/loyalty/i18n/vi.po [hunk 2]
```diff
@@ -1667,6 +1662,11 @@ msgid ""
 "program."
 msgstr ""
 
+#. module: loyalty
+#: model:loyalty.reward,description:loyalty.gift_card_program_reward
+msgid "Thẻ quà tặng"
+msgstr ""
```
Comment (tranngocson1996) [inline addons/loyalty/i18n/vi.po]: Chỗ này nhìn lạ vậy :D

**Comment by locdang8** [comment]
cc @hathu98

**Comment by binh1702** [comment]
@viinbot rebuild

**Pull request merged**